src/Dapr.Actors/Runtime/Actor.cs

```diff
@@ -348,15 +338,5 @@ internal void ValidateTimerCallback(ActorHost host, string callback)
                 throw new ArgumentException("Timer callback can only return type Task");
             }
         }
-
-        private void EnsureInteractorInitialized()
-        {
-            if (this.Host.DaprInteractor == null)
-            {
-                throw new InvalidOperationException(
-                    "The actor was initialized without an HTTP client, and so cannot interact with timers or reminders. " +
-                    "This is likely to happen inside a unit test.");
-            }
```

Comment:
Prior to this change we'd just tell you explicitly that this functionality can't be tested.